content/c-sharp/concepts/strings/terms/substring/substring.md

```diff
@@ -0,0 +1,96 @@
+---
+Title: '.Substring()'
+Description: 'Returns the substring of a string instance starting at a given index.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
```

Comment:
```suggestion
  - 'learn-c-sharp'
```
Include only those course/paths that are already part of catalog-content list, Here `learn-c-sharp` is not available in that list, so you can add it in the [catalog-content.md](https://github.com/Codecademy/docs/blob/main/documentation/catalog-content.md) file, so that it follows all the contribution guidelines.